### PR TITLE
fix(packages/fury-adapter-remote): implements `generateSourceMap`

### PR DESCRIPTION
**Pull request by marcofriso**

### CODE REVIEW

**Comment by kylef** [comment]
CI is broken due to https://github.com/apiaryio/api-elements.js/pull/395;

@marcofriso there is some strict linting on the commit messages so you'll need to change it to start with `fix(remote):`

**Comment by kylef** [comment]
https://github.com/apiaryio/api-elements.js/pull/395 is now merged; @marcofriso could you rebase this on master?

**Comment by marcofriso** [comment]
(there is a bug on Sourcetree that sometimes make the pull requests to close)

File: packages/fury-adapter-remote/package.json [hunk 1]
```diff
@@ -23,13 +23,13 @@
   },
   "dependencies": {
     "axios": "^0.19.0",
+    "chai": "^4.2.0",
```
Comment (kylef) [inline packages/fury-adapter-remote/package.json]: I think this change was accidental? chai is a devDependencies as it's only needed for the tests.

File: packages/fury-adapter-oas3-parser/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "test:integration:fix": "env GENERATE=1 mocha test/integration"
   },
   "dependencies": {
+    "chai": "^4.2.0",
```
Comment (kylef) [inline packages/fury-adapter-oas3-parser/package.json]: Ditto from previous comment.

File: packages/fury-adapter-remote/package.json (hunk 1 above)
Comment (marcofriso) [reply packages/fury-adapter-remote/package.json]: I received a warning somewhere I guess; changed

File: packages/fury-adapter-oas3-parser/package.json (hunk 2 above)
Comment (marcofriso) [reply packages/fury-adapter-oas3-parser/package.json]: changed

**Pull request merged**